src/widgetastic_patternfly5/charts/boxplot_chart.py

```diff
@@ -10,3 +10,174 @@ class BoxPlotChart(LineChart):
         id: If you want to look the input up by id, use this parameter, pass the id.
         locator: If you have specific locator else it will take pf-chart.
     """
+
+    def read(self, offset=(0, -100)):
+        """Read chart data.
+
+        Note: This method has some limitations as we are reading the tooltip for x-axis labels
+        with some offset. So only applicable for the chart which shows all Legend data in a single
+        tooltip for the respective x-axis label.
+
+        Args:
+            offset: offset to move the cursor from the x-axis label so that the tooltip can appear.
+        """
+        _data = {}
+
+        for lab_el in self._x_axis_labels_map.values():
+            self.browser.move_to_element(lab_el)
+            self.browser.click(lab_el)
+            self.browser.move_by_offset(*offset)
+            tooltip_el = self.browser.wait_for_element(self.TOOLTIP, timeout=5)
+
+            x_axis_label = self.browser.text(self.TOOLTIP_X_AXIS_LABLE, parent=tooltip_el)
+            # print("X axis label",x_axis_label)
+            label_data = {}
+
+            for label_el, value_el in zip(
+                self.browser.elements(self.TOOLTIP_LABLES, parent=tooltip_el),
+                self.browser.elements(self.TOOLTIP_VALUES, parent=tooltip_el),
+            ):
+                # print("Value is",self.browser.text(value_el))
+                # pdb.set_trace()
+                label = self.browser.text(label_el).strip()
+                if not label:
+                    label = "Data"
```

Comment:
We don't have to define any extra lable which is not available on UI. 
Embedded legend, actually showing extra things under that specific legend. 

in your case `Usage`. it should read like this

```
 '24:00': {
 	'Request': '2.3 cores',
 	'Limit': '2.3 cores',
 	'Usage': {
 		'Min: 1.48 cores,
 		'Max': 2.47 cores',
 		'Median': '2.25 cores',
 		'Q1': '2.17 cores',
 		'Q3': '2.38 cores'
 		}
	}
```